$NetBSD: patch-helpers_log_daemon_DB_config.test,v 1.1 2013/06/04 15:16:42 tron Exp $

Overright broken Perl config test that fails with Perl 5.18.0 and newer.

--- helpers/log_daemon/DB/config.test.orig	2013-05-20 12:48:55.000000000 +0100
+++ helpers/log_daemon/DB/config.test	2013-06-04 15:48:25.000000000 +0100
@@ -2,6 +2,7 @@
 
 ## Test: do we have perl to build the helper scripts?
 ## Test: do we have pod2man to build the manual?
-perl --version >/dev/null && echo | pod2man >/dev/null
+#perl --version >/dev/null && echo | pod2man >/dev/null
+true
 
 exit $?
